pytest.ini

```diff
@@ -1,5 +1,8 @@
 [pytest]
-DJANGO_SETTINGS_MODULE = hydroshare/settings.py
+addopts = -x --cov-report xml --cov-append --cov=pytest/functional --junitxml="pytests.xml"
```

Comment:
@cuahsimarko - are these pytest changes something we want to bring in?

Reply:
These should be vetted by Dan. Martin is peripherally involved for high-level oversight. I generally like to have everything parameterized per environment. For example, my laptop uses `-rx --no-migrations --reuse-db` via PyCharm. Pabitra can negotiate this with other devs and especially Dan. I'll go with whatever folks want.

Reply:
CUAHSI Jenkins may want to use other settings. Not sure if/how these are related to RENCU @cbcunc